pkg/operator/v1helpers/helpers.go

```diff
@@ -259,3 +261,18 @@ func (agg aggregate) Error() string {
 func (agg aggregate) Errors() []error {
 	return []error(agg)
 }
+
+func MapToEnvVars(proxyConfig map[string]string) []corev1.EnvVar {
```

Comment:
nit: would make this env var name something generic

Reply:
Yeah, I forgot about that :wink: